README.md

```diff
@@ -33,7 +33,19 @@ To test the project, you need first to install the server and get it running, th
 To install and start the server:
 
 1. Install Docker.
-   - [On Windows](https://docs.docker.com/docker-for-windows/install/).
+   - [On Windows 10 Pro](https://docs.docker.com/docker-for-windows/install/).
+   - [On Windows 10 Home and below](https://github.com/docker/toolbox/releases).
+      - Addition steps for Docker Toolbox(Windows 10 Home and below)
```

Comment:
The latest docker is not supported by windows 10 Home and below otherwise you would need to use the legacy "docker toolbox" which is why there are 2 different download links

Reply:
I understand, it's not that I'm talking about. You have a link that says it's for Windows 10 Home and below, and the following line repeats that.